docs/pages/svelte-kit.page.server.mdx

```diff
@@ -0,0 +1,10 @@
+import { Link } from '@brillout/docpress'
+import { MostlyMutations } from '../components'
```

Comment:
I believe we should also apply the `MostlyMutations` recommendation for SvelteKit. (The module `MostlyMutations` is impoted but not used.)